datalad/metadata/search.py

```diff
@@ -308,7 +308,13 @@ def matcher(s):
                 else:
                     report_dict = {}  # it was empty but not None -- asked to
                     # not report any specific field
-                yield opj(ds_path_prefix, location), report_dict
+                if isinstance(location, (list, tuple)):
+                    # could be that the same dataset installed into multiple
+                    # locations. For now report them separately
+                    for l in location:
+                        yield opj(ds_path_prefix, l), report_dict
```

Comment:
missing coverage there makes no sense since test did ran and locally that was needed to have it fixed... need to figure out WTF